potential_field_dmp/WayPoint[] waypoint